sample-input/benchmarks/c5g7/c5g7-cmfd.py

```diff
@@ -318,28 +319,29 @@
 track_generator.setNumThreads(num_threads)
 track_generator.generateTracks()
 
+
 ###############################################################################
-###########################   Running a Simulation   ##########################
+#                            Running a Simulation
 ###############################################################################
 
-solver = CPUSolver(geometry, track_generator)
-solver.setSourceConvergenceThreshold(tolerance)
+solver = CPUSolver(track_generator)
+solver.setConvergenceThreshold(tolerance)
 solver.setNumThreads(num_threads)
-solver.convergeSource(max_iters)
+solver.computeEigenvalue(max_iters)
```

Comment:
Are you asking a question, or stating your opinion that the method signature isn't descriptive enough? If it is not, any ideas on a better name?